### PR TITLE
Issue 3535: clusterctl get kubeconfig docs added

### LINKED ISSUE #3535: Add documentation for clusterctl sub-command `get kubeconfig`
**What steps did you take and what happened:**
[A clear and concise description on how to REPRODUCE the bug.]
Clusterctl added subcomand `get kubeconfig` command via #3460 .
Although the documentation for the same is missing in the cluster-api book.

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster-api version:
- Minikube/KIND version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

/kind documentation
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

### PR DESCRIPTION
**Pull request by iamemilio**
:book:

**What this PR does / why we need it**: Adds documentation clusterctl get kubeconfig command

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3535

/cc @prankul88 @fabriziopandini  @wfernandes

### CODE REVIEW

File: docs/book/src/clusterctl/commands/get-kubeconfig.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# clusterctl get kubeconfig
+
+The `clusterctl get` command prints the kubeconfig of an existing workload cluster into stdout.
```
Comment (wfernandes) [inline docs/book/src/clusterctl/commands/get-kubeconfig.md:3]: There could be more sub-commands in the future under the `get` verb.
```suggestion
This command prints the kubeconfig of an existing workload cluster into stdout.
```

File: docs/book/src/clusterctl/commands/get-kubeconfig.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+This funcitonality is available in clusterctl version 0.3.9 or newer.
```
Comment (wfernandes) [inline docs/book/src/clusterctl/commands/get-kubeconfig.md:4]:
```suggestion
This functionality is available in clusterctl v0.3.9 or newer.
```

**Review by wfernandes — COMMENTED** [review]
Thanks for this PR!
When I clicked on the link in the quickstart, the page 404'd for me. So not sure what's going on there.

Also add a reference to the [commands page](https://github.com/kubernetes-sigs/cluster-api/blob/0db3effb8ab0655d781090f9afba441bf66e6cf1/docs/book/src/clusterctl/commands/commands.md) and the [index page](https://github.com/kubernetes-sigs/cluster-api/blob/0db3effb8ab0655d781090f9afba441bf66e6cf1/docs/book/src/SUMMARY.md).

**Comment by prankul88** [comment]
Thankyou for the PR!
Adding to Warren's point, the command should also be referenced in the [overview page](https://github.com/kubernetes-sigs/cluster-api/blob/0db3effb8ab0655d781090f9afba441bf66e6cf1/docs/book/src/clusterctl/overview.md) .

**Review by prankul88 — COMMENTED** [review]
Thankyou for the PR!
Adding to Warren's point, the command should also be referenced in the [overview page](https://github.com/kubernetes-sigs/cluster-api/blob/0db3effb8ab0655d781090f9afba441bf66e6cf1/docs/book/src/clusterctl/overview.md).

File: docs/book/src/clusterctl/commands/get-kubeconfig.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Usage:
```
Comment (fabriziopandini) [inline docs/book/src/clusterctl/commands/get-kubeconfig.md:6]:
```suggestion
```
For sake of consistency with others command's doc, we should drop Usage.
Also, I suggest to drop the list of parameters; instead, you can provide an example of calling the command with the `--namespace` flag

Comment (iamemilio) [reply docs/book/src/clusterctl/commands/get-kubeconfig.md:6]: ack

File: docs/book/src/SUMMARY.md [hunk 4]
```diff
@@ -20,6 +20,7 @@
         - [init](clusterctl/commands/init.md)
         - [config cluster](clusterctl/commands/config-cluster.md)
         - [generate yaml](clusterctl/commands/generate-yaml.md)
+        - [clusterctl get kubeconfig](clusterctl/commands/get-kubeconfig.md)
```
Comment (wfernandes) [inline docs/book/src/SUMMARY.md:23]: nit: For consistency
```suggestion
        - [get kubeconfig](clusterctl/commands/get-kubeconfig.md)
```

**Review by wfernandes — COMMENTED** [review]
Lgtm except for the one nit.

**Comment by prankul88** [comment]
LGTM
Thanks

**Comment by fabriziopandini** [comment]
/approve 
pending fix for latest @wfernandes comment

**Pull request merged**